integration/run_test.go

```diff
@@ -278,7 +278,13 @@ func setUp(t *testing.T) (kubernetes.Interface, *v1.Namespace, func(t *testing.T
 		t.Fatalf("install: %v", err)
 	}
 
-	createAttestationAuthority(t, *gcpProject, ns.Name)
+	// Generate a key value pair
+	pubKey, privKey := testutil.CreateKeyPair(t, aaSecret)
```

Comment:
I think 'aaSecret' can go.  Just write 'test-attestor' here and comment that it must be the same as in my-isp.yaml.

Reply:
It's also used in "cleanup.go". so it might still be easier to centralize it?

Reply:
Oh, OK, I didn't know that.  Then let's leave it.